content/intro-to-storybook/react/en/screen.md

```diff
@@ -147,7 +164,7 @@ We also need to change the `App` component to render the `InboxScreen` (eventual
 + import store from './lib/store';
 
 + import { Provider } from 'react-redux';
-+ import { InboxScreen } from './components/InboxScreen';
++ import InboxScreen from './components/InboxScreen';
 
 function App() {
   return (
```

Comment:
This is invalid JSX (or maybe just React, I don't remember). You must return a single element, i.e. wrap these two elements in a fragment (`<>`).